RHIDP-6507 - modularize _Monitoring and logging with AWS (#1089)
redhat-developer/red-hat-developers-documentation-rhdh: modules/observe/proc-retrieving-logs-from-amazon-cloudwatch.adoc

[id="retrieving-logs-from-amazon-cloudwatch_{context}"]
= Retrieving logs from Amazon CloudWatch

.Prerequisites

> * CloudWatch Container Insights is used to capture logs and metrics for Amazon EKS.

Author: ```suggestion
* CloudWatch Container Insights is used to capture logs and metrics for {eks-brand-name}.
```

> * The CloudWatch Container Insights are used to capture logs and metrics for Amazon EKS.

Reviewer: ```suggestion
* CloudWatch Container Insights is used to capture logs and metrics for Amazon EKS.
```
Consider adding the Amazon or EKS attribute in front of CloudWatch for proper branding. 

CloudWatch Container Insights ends with an "s" but it is a single tool, therefore, it is correct to use "is" instead of "are". Also, AWS docs doesn't usually use "The" in front, so I removed it to align with their branding.

> For more information, see https://docs.aws.amazon.com/prescriptive-guidance/latest/implementing-logging-monitoring-cloudwatch/kubernetes-eks-logging.html[Logging for Amazon EKS] documentation.

Reviewer: Consider using the attribute for Amazon EKS

Author: ```suggestion
For more information, see https://docs.aws.amazon.com/prescriptive-guidance/latest/implementing-logging-monitoring-cloudwatch/kubernetes-eks-logging.html[Logging for {eks-brand-name}] documentation.
```

> * To capture the logs and metrics, https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-EKS-addon.html[install the Amazon CloudWatch Observability EKS add-on] in your cluster.

Reviewer: ```suggestion
* To capture the logs and metrics, link:https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-EKS-addon.html[install the Amazon CloudWatch Observability EKS add-on] in your cluster.
```

Reviewer: Also consider replacing EKS with attribute

Following the setup of Container Insights, you can access container logs using Logs Insights or Live Tail views.

* CloudWatch names the log group where all container logs are consolidated in the following manner:

> +

Author: ```suggestion
+
[subs="+quotes"]
```

----

> /aws/containerinsights/<ClusterName>/application

Reviewer: ```suggestion
/aws/containerinsights/_<ClusterName>_/application
```
To italicize the variable per CCS guidelines

----

.Procedure

> * Following is an example query to retrieve logs from the Developer Hub instance:

Reviewer: Replace Developer Hub with attribute

Author: ```suggestion
* Following is an example query to retrieve logs from the {product-short} instance:
```

[id="retrieving-logs-from-amazon-cloudwatch_{context}"]
= Retrieving logs from Amazon CloudWatch

.Prerequisites
* CloudWatch Container Insights is used to capture logs and metrics for {eks-brand-name}.
For more information, see https://docs.aws.amazon.com/prescriptive-guidance/latest/implementing-logging-monitoring-cloudwatch/kubernetes-eks-logging.html[Logging for {eks-brand-name}] documentation.

* To capture the logs and metrics, link:https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-EKS-addon.html[install the Amazon CloudWatch Observability EKS add-on] in your cluster.
Following the setup of Container Insights, you can access container logs using Logs Insights or Live Tail views.

* CloudWatch names the log group where all container logs are consolidated in the following manner:
+
[subs="+quotes"]
----
/aws/containerinsights/_<cluster_name>_/application
----

.Procedure

> * Following is an example query to retrieve logs from the {product-short} instance:

Reviewer: Convert this to an actionable procedural step. Currently, this is a statement rather than a step or action that the user can do.

Author: ```suggestion
* To retrieve logs from the {product-short} instance, run a query such as:
```
